src/forwarders.js

```diff
@@ -440,6 +443,7 @@ export default function Forwarders(mpInstance, kitBlocker) {
         });
     };
 
+    // TODO: This is a v1 method, we should deprecate
```

Comment:
Please instead make this a ticket and add the ticket instead of this description.